File: 059- Sum Of Binary Digits/SumOfBinaryDigits.py
# Function to compute sum of binary digits
def sum_of_binary_digits(n):
    binary_str = bin(n)[2:]  # Convert to binary and remove '0b'
    return sum(int(digit) for digit in binary_str)


n = 15
print("Sum of binary digits:", sum_of_binary_digits(n))

File: 059- Sum Of Binary Digits/SumOfBinaryDigits.cpp
#include<iostream>
using namespace std;

int SumofBinaryDigits(int n) {
    int sum = 0;
    while (n > 0) {
        sum += n % 2; // Add the last binary digit
        n /= 2;       // Remove the last binary digit
    }
    return sum;
}

int main() {
    int n;
    cout << "Enter a number: ";
    cin >> n;
    int result = SumofBinaryDigits(n);
    cout << "Sum of binary digits: " << result << endl;
    return 0;
}